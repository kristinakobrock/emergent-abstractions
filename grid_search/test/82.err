  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 16.07it/s] 10%|#########3                                                                                   | 5/50 [00:00<00:02, 22.32it/s] 16%|##############8                                                                              | 8/50 [00:00<00:01, 23.47it/s] 22%|####################2                                                                       | 11/50 [00:00<00:01, 25.43it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 25.00it/s] 34%|###############################2                                                            | 17/50 [00:00<00:01, 24.68it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 24.70it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 25.81it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 25.87it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 29.45it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.80it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 28.59it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 27.51it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 26.53it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 24.09it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.11it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.55it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 20.61it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 22.77it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 17.64it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 18.10it/s]100%|############################################################################################| 13/13 [00:00<00:00, 19.37it/s]
