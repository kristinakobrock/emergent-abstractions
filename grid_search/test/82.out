# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6873588562011719, "acc": 0.5906543135643005, "length": 3.4885997772216797, "mode": "train", "epoch": 1}
{"loss": 0.6790875792503357, "acc": 0.5910155773162842, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6661161780357361, "acc": 0.6199864745140076, "length": 3.3329269886016846, "mode": "train", "epoch": 2}
{"loss": 0.6766108274459839, "acc": 0.5992187857627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6620911955833435, "acc": 0.6239388585090637, "length": 3.101562738418579, "mode": "train", "epoch": 3}
{"loss": 0.6684583425521851, "acc": 0.6263671517372131, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6555688381195068, "acc": 0.6247933506965637, "length": 2.9505977630615234, "mode": "train", "epoch": 4}
{"loss": 0.6435391306877136, "acc": 0.6374603509902954, "length": 1.90234375, "mode": "test", "epoch": 4}
{"loss": 0.6325199604034424, "acc": 0.6423711776733398, "length": 3.124020576477051, "mode": "train", "epoch": 5}
{"loss": 0.6127769947052002, "acc": 0.6673492193222046, "length": 3.49609375, "mode": "test", "epoch": 5}
{"loss": 0.59123694896698, "acc": 0.6912350654602051, "length": 3.442211627960205, "mode": "train", "epoch": 6}
{"loss": 0.586483895778656, "acc": 0.7012878656387329, "length": 3.27734375, "mode": "test", "epoch": 6}
{"loss": 0.5580483675003052, "acc": 0.7185590267181396, "length": 3.348790407180786, "mode": "train", "epoch": 7}
{"loss": 0.5702333450317383, "acc": 0.7172729969024658, "length": 3.6875, "mode": "test", "epoch": 7}
{"loss": 0.5296178460121155, "acc": 0.7361005544662476, "length": 3.620880365371704, "mode": "train", "epoch": 8}
{"loss": 0.5688143372535706, "acc": 0.7107237577438354, "length": 3.99609375, "mode": "test", "epoch": 8}
{"loss": 0.5107530355453491, "acc": 0.7477206587791443, "length": 3.817861557006836, "mode": "train", "epoch": 9}
{"loss": 0.572296679019928, "acc": 0.7245880365371704, "length": 3.98046875, "mode": "test", "epoch": 9}
{"loss": 0.49197980761528015, "acc": 0.7614992260932922, "length": 3.756256103515625, "mode": "train", "epoch": 10}
{"loss": 0.5729795098304749, "acc": 0.729934811592102, "length": 3.92578125, "mode": "test", "epoch": 10}
{"loss": 0.47937026619911194, "acc": 0.7669376730918884, "length": 3.7739624977111816, "mode": "train", "epoch": 11}
{"loss": 0.55320805311203, "acc": 0.7333495616912842, "length": 3.94921875, "mode": "test", "epoch": 11}
{"loss": 0.46994006633758545, "acc": 0.7757509350776672, "length": 3.7782607078552246, "mode": "train", "epoch": 12}
{"loss": 0.5478208065032959, "acc": 0.7382811903953552, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.4583519399166107, "acc": 0.7846857309341431, "length": 3.8945212364196777, "mode": "train", "epoch": 13}
{"loss": 0.5478867292404175, "acc": 0.7484374642372131, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.44689416885375977, "acc": 0.7888519167900085, "length": 3.944705009460449, "mode": "train", "epoch": 14}
{"loss": 0.5582008361816406, "acc": 0.7480469346046448, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.4363217353820801, "acc": 0.7942214608192444, "length": 3.9710066318511963, "mode": "train", "epoch": 15}
{"loss": 0.5399899482727051, "acc": 0.7525390982627869, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.42508822679519653, "acc": 0.8015536069869995, "length": 3.9819436073303223, "mode": "train", "epoch": 16}
{"loss": 0.5457227230072021, "acc": 0.7470703125, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.421292781829834, "acc": 0.8020603060722351, "length": 3.9807045459747314, "mode": "train", "epoch": 17}
{"loss": 0.5405623316764832, "acc": 0.7539063096046448, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4150584936141968, "acc": 0.8059027791023254, "length": 3.9870760440826416, "mode": "train", "epoch": 18}
{"loss": 0.5182099938392639, "acc": 0.76171875, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.40803611278533936, "acc": 0.8089537620544434, "length": 3.9891865253448486, "mode": "train", "epoch": 19}
{"loss": 0.5285143852233887, "acc": 0.7560546398162842, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4006010890007019, "acc": 0.8153023719787598, "length": 3.988323211669922, "mode": "train", "epoch": 20}
{"loss": 0.5130031108856201, "acc": 0.7693358063697815, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3955381512641907, "acc": 0.8183723092079163, "length": 3.9914305210113525, "mode": "train", "epoch": 21}
{"loss": 0.5066575407981873, "acc": 0.7746093273162842, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.3887459635734558, "acc": 0.8221535682678223, "length": 3.9928996562957764, "mode": "train", "epoch": 22}
{"loss": 0.49511682987213135, "acc": 0.7828125357627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3858897089958191, "acc": 0.8257136940956116, "length": 3.9895262718200684, "mode": "train", "epoch": 23}
{"loss": 0.5116009712219238, "acc": 0.7748046517372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3796404004096985, "acc": 0.8288190364837646, "length": 3.9949424266815186, "mode": "train", "epoch": 24}
{"loss": 0.480632483959198, "acc": 0.7845702171325684, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3734431564807892, "acc": 0.8316643238067627, "length": 3.9944040775299072, "mode": "train", "epoch": 25}
{"loss": 0.48671332001686096, "acc": 0.7767577767372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3670244514942169, "acc": 0.8344222903251648, "length": 3.996222972869873, "mode": "train", "epoch": 26}
{"loss": 0.4832947254180908, "acc": 0.7802734375, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.35985055565834045, "acc": 0.8378900289535522, "length": 3.9972546100616455, "mode": "train", "epoch": 27}
{"loss": 0.474193274974823, "acc": 0.7857422232627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.35668066143989563, "acc": 0.8409878611564636, "length": 3.997218370437622, "mode": "train", "epoch": 28}
{"loss": 0.45937561988830566, "acc": 0.7882813811302185, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.35275933146476746, "acc": 0.8440234065055847, "length": 3.9972901344299316, "mode": "train", "epoch": 29}
{"loss": 0.46442124247550964, "acc": 0.7955078482627869, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3500972390174866, "acc": 0.846159040927887, "length": 3.997744560241699, "mode": "train", "epoch": 30}
{"loss": 0.4552081227302551, "acc": 0.7923827767372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.34560588002204895, "acc": 0.8476693034172058, "length": 3.9983718395233154, "mode": "train", "epoch": 31}
{"loss": 0.4483782947063446, "acc": 0.7978516221046448, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3391413986682892, "acc": 0.8537825345993042, "length": 3.9973433017730713, "mode": "train", "epoch": 32}
{"loss": 0.44068336486816406, "acc": 0.8095702528953552, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3335755169391632, "acc": 0.8550911545753479, "length": 3.999559164047241, "mode": "train", "epoch": 33}
{"loss": 0.43020206689834595, "acc": 0.8175781965255737, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3269801735877991, "acc": 0.8580588698387146, "length": 3.9986588954925537, "mode": "train", "epoch": 34}
{"loss": 0.43867021799087524, "acc": 0.8068360686302185, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3245636224746704, "acc": 0.8629137873649597, "length": 3.999128580093384, "mode": "train", "epoch": 35}
{"loss": 0.43285149335861206, "acc": 0.8166016340255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3219389319419861, "acc": 0.8608542084693909, "length": 3.999364137649536, "mode": "train", "epoch": 36}
{"loss": 0.43346846103668213, "acc": 0.8132811784744263, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.31911298632621765, "acc": 0.8668418526649475, "length": 3.9976418018341064, "mode": "train", "epoch": 37}
{"loss": 0.456221342086792, "acc": 0.8185547590255737, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3098183870315552, "acc": 0.8701794147491455, "length": 3.9987597465515137, "mode": "train", "epoch": 38}
{"loss": 0.42431551218032837, "acc": 0.8218750357627869, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.30650800466537476, "acc": 0.8719525933265686, "length": 3.9994518756866455, "mode": "train", "epoch": 39}
{"loss": 0.43001651763916016, "acc": 0.8294921517372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.30152618885040283, "acc": 0.8769938349723816, "length": 3.999166488647461, "mode": "train", "epoch": 40}
{"loss": 0.40995723009109497, "acc": 0.8298827409744263, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2894035875797272, "acc": 0.8794931173324585, "length": 3.9961678981781006, "mode": "train", "epoch": 41}
{"loss": 0.40490952134132385, "acc": 0.8373046517372131, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.29071712493896484, "acc": 0.8793893456459045, "length": 3.9977304935455322, "mode": "train", "epoch": 42}
{"loss": 0.4214213490486145, "acc": 0.8337889313697815, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2957184314727783, "acc": 0.8785527944564819, "length": 3.998349905014038, "mode": "train", "epoch": 43}
{"loss": 0.42839503288269043, "acc": 0.8357421159744263, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2834264636039734, "acc": 0.8868975639343262, "length": 3.9970169067382812, "mode": "train", "epoch": 44}
{"loss": 0.39412301778793335, "acc": 0.8460936546325684, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2862968146800995, "acc": 0.8830583691596985, "length": 3.9993796348571777, "mode": "train", "epoch": 45}
{"loss": 0.4030567407608032, "acc": 0.8466796278953552, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.27947357296943665, "acc": 0.8893758058547974, "length": 3.9997329711914062, "mode": "train", "epoch": 46}
{"loss": 0.3953958749771118, "acc": 0.8542967438697815, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.26926907896995544, "acc": 0.8922300338745117, "length": 3.999865770339966, "mode": "train", "epoch": 47}
{"loss": 0.3904555141925812, "acc": 0.8525390625, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.2622555196285248, "acc": 0.8962351679801941, "length": 3.9998714923858643, "mode": "train", "epoch": 48}
{"loss": 0.3889736831188202, "acc": 0.8542968034744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2621637284755707, "acc": 0.8999342918395996, "length": 3.9998080730438232, "mode": "train", "epoch": 49}
{"loss": 0.38874536752700806, "acc": 0.8488281965255737, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2586368918418884, "acc": 0.8998100757598877, "length": 3.9995152950286865, "mode": "train", "epoch": 50}
{"loss": 0.40917402505874634, "acc": 0.8427734375, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.25195011496543884, "acc": 0.9039145112037659, "length": 3.998504161834717, "mode": "train", "epoch": 51}
{"loss": 0.3810698986053467, "acc": 0.8525390625, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.25266873836517334, "acc": 0.901515543460846, "length": 3.9999027252197266, "mode": "train", "epoch": 52}
{"loss": 0.3816359043121338, "acc": 0.8542967438697815, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.23949773609638214, "acc": 0.9085006713867188, "length": 3.996185541152954, "mode": "train", "epoch": 53}
{"loss": 0.3713977038860321, "acc": 0.8587890267372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2408517748117447, "acc": 0.9097660183906555, "length": 3.99985671043396, "mode": "train", "epoch": 54}
{"loss": 0.4033697545528412, "acc": 0.8513670563697815, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.24040541052818298, "acc": 0.90743488073349, "length": 3.9982457160949707, "mode": "train", "epoch": 55}
{"loss": 0.3981889486312866, "acc": 0.8535155653953552, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.23489424586296082, "acc": 0.9113411903381348, "length": 3.9997262954711914, "mode": "train", "epoch": 56}
{"loss": 0.3903718590736389, "acc": 0.8548827767372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.23628582060337067, "acc": 0.9131610989570618, "length": 3.9997451305389404, "mode": "train", "epoch": 57}
{"loss": 0.3801307678222656, "acc": 0.8628906011581421, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2331521213054657, "acc": 0.911819338798523, "length": 3.9993741512298584, "mode": "train", "epoch": 58}
{"loss": 0.3896128237247467, "acc": 0.8498047590255737, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.22685003280639648, "acc": 0.9151080250740051, "length": 3.999941349029541, "mode": "train", "epoch": 59}
{"loss": 0.39460957050323486, "acc": 0.853515625, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.24109181761741638, "acc": 0.9092773795127869, "length": 3.999486207962036, "mode": "train", "epoch": 60}
{"loss": 0.3890399634838104, "acc": 0.8597655892372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7442646622657776
